pkg/bitwarden/cli_test.go

```diff
@@ -0,0 +1,561 @@
+package bitwarden
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestNewCliClientWithRun(t *testing.T) {
+	testCases := []struct {
+		name               string
+		username           string
+		password           string
+		run                func(cmd string, args ...string) ([]byte, error)
+		expectedSession    string
+		expectedSavedItems []Item
+		expectedError      error
+	}{
+		{
+			name:     "basic case",
+			username: "u",
+			password: "p",
+			run: func(cmd string, args ...string) ([]byte, error) {
```

Comment:
I would much rather see a simple test fake that has a map of args to responses instead of this

Reply:
Stole your testing code.